.github/workflows/push.yml

```diff
@@ -94,101 +93,6 @@ jobs:
           name: '${{env.CODECOV_UNIQUE_NAME}}'  
             
             
-  Generate-Documentation:
-    name: Generate Documentation
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/develop'
-    needs: Push-Workflow
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-        # with:
-        #   persist-credentials: false
-      - name: Generate Documentation of HTML pages
-        run: |
-          npm install --global typedoc
-          npm install typedoc-plugin-markdown
-          npm i --save-dev @types/node
-          npx typedoc --entryPoints src --out talawa-api-docs --plugin typedoc-plugin-markdown --theme markdown --entryPointStrategy expand
-      
-      - name: Make Markdown Files MDX Compatible
-        run: python3 .github/workflows/md_mdx_format_adjuster.py --directory talawa-api-docs 
```

Comment:
Please remove the `./.github/workflows/md_mdx_format_adjuster.py`  script if it's not being used any more